### PR TITLE
#596 - Show transcription source labels in admin

### PR DESCRIPTION
**Pull request by blms**
## What this PR does

- Per #596:
  - When displaying multiple transcriptions in the document admin: groups transcriptions by source, labels them each with source citation

### CODE REVIEW

File: geniza/corpus/templates/admin/corpus/document/change_form.html [hunk 1]
```diff
@@ -18,11 +18,21 @@
...
     {% if original.has_transcription %}
-        <h2>Transcription</h2>
-        {% for footnote in original.footnotes.all %}
-            {% if footnote.content %}
-                {% include "footnotes/transcription.html" with transcription=footnote.content %}
-            {% endif %}
-        {% endfor %}
+        <fieldset class="module aligned transcriptions-field">
+            <div class="form-row">
+                <label>Transcription(s)</label>
```
Comment (rlskoeser) [inline geniza/corpus/templates/admin/corpus/document/change_form.html:23]: let's use `pluralize` here

File: geniza/corpus/templates/admin/corpus/document/change_form.html [hunk 2]
```diff
@@ -18,11 +18,21 @@
...
+                {% regroup original.footnotes.all by source as source_list %}
```
Comment (rlskoeser) [inline geniza/corpus/templates/admin/corpus/document/change_form.html:24]: can't we just iterate over our `digital_editions` method? and I don't think we should regroup (don't need to, but also better to be explicit about what is attached where e.g. when transcription content needs to be consolidated in TEI after merging documents)

File: geniza/corpus/templates/admin/corpus/document/change_form.html [hunk 3]
```diff
@@ -18,11 +18,21 @@
...
+                {% for source in source_list %}
+                    {% if source_list|length > 1 %}<h3 class="source">{{ source.grouper|safe }}</h3>{% endif %}
+                    {% for footnote in source.list %}
+                        {% if footnote.content %}
+                            {% include "footnotes/transcription.html" with transcription=footnote.content %}
+                        {% else %}
+                            <div class="transcription">No transcription</div>
```
Comment (rlskoeser) [inline geniza/corpus/templates/admin/corpus/document/change_form.html:31]: I don't think we should display this here — it's redundant and takes up space.

There's already a digitized transcription red/green boolean indicator in the footnote inline display; if that isn't enough for the team (in combination with labels for the transcriptions), then we should figure out what would make it more readable for them.

**Review by rlskoeser — CHANGES_REQUESTED** [review]
looks good, just requesting a couple adjustments

File: geniza/corpus/templates/admin/corpus/document/change_form.html (hunk 1 above)
Comment (blms) [reply geniza/corpus/templates/admin/corpus/document/change_form.html]: I always forget about that! Done in c62ff7a 👍

File: geniza/corpus/templates/admin/corpus/document/change_form.html (hunk 2 above)
Comment (blms) [reply geniza/corpus/templates/admin/corpus/document/change_form.html]: Done in c62ff7a 👍

File: geniza/corpus/templates/admin/corpus/document/change_form.html (hunk 3 above)
Comment (blms) [reply geniza/corpus/templates/admin/corpus/document/change_form.html]: Done in c62ff7a 👍

**Review by rlskoeser — APPROVED** [review]

**Pull request merged**